File: Software Technologies/Exam Prep III - IMDB/JavaScript Skeleton/models/Film.js
const mongoose = require('mongoose');

let filmSchema = mongoose.Schema({
    name: {type: String, required: true},
    genre: {type: String, required: true},
    director: {type: String, required: true},
    year: {type: Number, required: true}
});

let Film = mongoose.model('Film', filmSchema);

module.exports = Film;

File: Software Technologies/Exam Prep III - IMDB/C# Skeleton/IMDB/Models/Film.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace IMDB.Models
{
    public class Film
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [AllowHtml]
        [StringLength(50)]
        public string Name { get; set; }

        [Required]
        [AllowHtml]
        public string Genre { get; set; }

        [Required]
        [AllowHtml]
        public string Director { get; set; }

        [Required]
        public int Year { get; set; }
    }
}